lib/packwerk/commands/uses_parse_run.rb

```diff
@@ -0,0 +1,91 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "optparse"
+
+module Packwerk
+  module Commands
+    module UsesParseRun
+      extend T::Sig
+      extend T::Helpers
+
+      requires_ancestor { BaseCommand }
+
+      sig do
+        params(
+          args: T::Array[String],
+          configuration: Configuration,
+          out: T.any(StringIO, IO),
+          err_out: T.any(StringIO, IO),
+          progress_formatter: Formatters::ProgressFormatter,
+          offenses_formatter: OffensesFormatter,
+        ).void
+      end
+      def initialize(args, configuration:, out:, err_out:, progress_formatter:, offenses_formatter:)
+        @args = args
+        @configuration = configuration
```

Comment:
These should be assigned by super, right?

Reply:
https://github.com/Shopify/packwerk/pull/351/commits/2d9c5c02d34ba7f90df9c65a2f4bf21637e70973#diff-7a35006173631f73820a5c2882b286cbe58356933b7e74a50c5446d1c1547ce9L25

Reply:
Hm, I wonder if this is a bug in Sorbet. If we require the class as an ancestor, it should always expect the initializer to run.

Reply:
But it doesn't know what any other initializer might run in the meantime? (If for example more modules are included.)

Reply:
True, I guess it can't know for sure. The workaround is good 👍